File: frontend/src/models/DetalleSemillasTetrazolioDto.ts
export interface DetalleSemillasTetrazolioDto {
  id: number | null;
  tetrazolioId: number | null;
  numeroTabla: number;

  vsd_total: number; vsd_mecanico: number; vsd_ambiente: number; vsd_chinches: number; vsd_fracturas: number; vsd_otros: number; vsd_duras: number;
  vl_total: number;  vl_mecanico: number;  vl_ambiente: number;  vl_chinches: number;  vl_fracturas: number;  vl_otros: number;  vl_duras: number;
  vm_total: number;  vm_mecanico: number;  vm_ambiente: number;  vm_chinches: number;  vm_fracturas: number;  vm_otros: number;  vm_duras: number;
  vs_total: number;  vs_mecanico: number;  vs_ambiente: number;  vs_chinches: number;  vs_fracturas: number;  vs_otros: number;  vs_duras: number;
  nv_total: number;  nv_mecanico: number;  nv_ambiente: number;  nv_chinches: number;  nv_fracturas: number;  nv_otros: number;  nv_duras: number;

  activo: boolean;
}

File: src/main/java/ti/proyectoinia/dtos/DetalleSemillasTetrazolioDto.java
package ti.proyectoinia.dtos;

import lombok.Data;

@Data
public class DetalleSemillasTetrazolioDto {
    private Long id;
    private Long tetrazolioId;
    private Integer numeroTabla;

    public Integer vsd_total, vsd_mecanico, vsd_ambiente, vsd_chinches, vsd_fracturas, vsd_otros, vsd_duras;
    public Integer vl_total,  vl_mecanico,  vl_ambiente,  vl_chinches,  vl_fracturas,  vl_otros,  vl_duras;
    public Integer vm_total,  vm_mecanico,  vm_ambiente,  vm_chinches,  vm_fracturas,  vm_otros,  vm_duras;
    public Integer vs_total,  vs_mecanico,  vs_ambiente,  vs_chinches,  vs_fracturas,  vs_otros,  vs_duras;
    public Integer nv_total,  nv_mecanico,  nv_ambiente,  nv_chinches,  nv_fracturas,  nv_otros,  nv_duras;

    private boolean activo;
}


